index.html

```diff
@@ -3074,94 +3078,76 @@ <h3>WoT Binding Templates</h3>
+                <p>
+                  <a>Transport protocols</a> may have extension mechanisms that must be known to interact successfully.
+                  Such <a>subprotocols</a> cannot be identified from the URI scheme alone and must be declared explicitly
+                  using the <code>subprotocol</code> field inside forms.
+                  Examples for HTTP are long polling [[?RFC6202]] (<code>longpoll</code>) or 
+                  Server-Sent Events [[?HTML]] (<code>sse</code>).
+                </p>
+            </li>
+          </ul>
         </li>
         <li>
-          <b>Subprotocol:</b>
+          <b>Payload Representation</b>
           <p>
-            <a>Transfer protocols</a> may have extension mechanisms that must be known to interact successfully.
-            Such <a>subprotocols</a> cannot be identified from the URI scheme alone and must be declared explicitly.
-            Examples are the push notification workarounds for HTTP such as long polling [[?RFC6202]] or Server-Sent
-            Events [[?HTML]].
-            Forms may contain the necessary information to identify the <a>subprotocol</a> in additional form fields.
+            Representation formats (a.k.a. serializations) used for exchanging data can differ between protocols, 
+            <a>IoT Platforms</a> and standards.
+            The media type [[!RFC2046]] identifies these formats, while media type parameters may specify them further.
+            Forms may contain the media type and optional parameters in additional form fields such as a content type 
+            field known from HTTP, which combines media type and its potential parameters
+            (e.g., <code>text/plain; charset=utf-8</code>).
           </p>
         </li>
         <li>
-          <b>Security:</b>
+          <b>Platforms and Standards</b>
           <p>
-            Security mechanisms can be applied at different layers of the communication stack
-            and might be used together, often to complement each other.
-            Examples are (D)TLS [[?RFC8446]]/[[?RFC6347]],
-            IPSec [[?RFC4301]],
-            OAuth [[?RFC6749]], and
-            ACE [[?RFC7744]].
-            Due to the cross-cutting nature of security, the necessary information to apply the right mechanism
-            may be given within the general metadata of the <a>Thing</a>
-            and/or specialized for each <a>Interaction Affordance</a> or form.
+            <a>IoT Platforms</a> and standards often introduce proprietary modifications at the application layer such as 
+            platform-specific HTTP header fields or CoAP options. 
+            They can also contain specific payload structures on top of the media types that need to be accurately 
+            incorporated into each message between the Thing and its Consumer.
+            Together with the protocol modification, payload representation specific to different platforms can be represented
```

Comment:
grammar

Reply:
not sure what was the issue but I have reformulated a bit

Reply:
No replies so I guess it is fine